collector/collector.go

```diff
@@ -230,12 +271,20 @@ func (c collector) Collect(ch chan<- prometheus.Metric) {
 		prometheus.NewDesc("snmp_scrape_walk_duration_seconds", "Time SNMP walk/bulkwalk took.", nil, nil),
 		prometheus.GaugeValue,
 		time.Since(start).Seconds())
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc("snmp_scrape_packets_sent", "Packets sent for the walk.", nil, nil),
```

Comment:
It's not clear from the help text if you're counting the retries.

Reply:
Hrm, good question, I would have to check the library code to see if that's included as well.

Reply:
I reviewed the SNMP library, and retries are included in packets sent.